src/handler/BallotHandler.ts

```diff
@@ -47,4 +59,52 @@ export class BallotHandler {
 
     await ballotService.updateBallot(ballotID, running, question, options);
   }
+
+  public async getAdminStatus(
```

Comment:
Choose a better name like `getVoteResult`

Reply:
Sorry cannot select lines on mobile, but most of this functions should be in a service, see other comment.

Reply:
Renamed in [#54 Moved counting logic to voting service and refactored](https://github.com/LeoTuet/backend-we-want-to-sleep/pull/54/commits/d2c6e6ec6461eb9eec43029b972964306cbd4c12)